src/runtime_src/core/tools/xbutil2/SubCmdReset.cpp

```diff
@@ -45,7 +46,39 @@ pretty_print_action_list(xrt_core::device_collection& deviceCollection, xrt_core
 static void
 reset_device(std::shared_ptr<xrt_core::device> dev, xrt_core::query::reset_type reset)
 {
-  dev->reset(reset);
+  if (xrt_core::device_query<xrt_core::query::rom_vbnv>(dev).find("_u30_") != std::string::npos) {
+    /*
```

Comment:
For comments like this, it is better to use C++ comments (e.g., `//`) and not C comments (e.g., `/* */`).

The reasoning behind this thinking has to do with development.  There are times when developers wish to comment out a section of code and they use C comments to do so.  Unfortunately, nested C comments are supported only with a few compilers.

Note: It could be argued that the developer use a `#if 0   #endif` statement.

Reply:
Done